### ISSUE TITLE
Serene Season New update causes minecraft 1.20.1 to crash (Sorry If its an older version)

### ISSUE THREAD

**Issue by Clyd3Th3R4bb1t** [Glitchfiend/SereneSeasons#498]
### What's the issue you encountered?

If i tried to load up a world with the new Update it freezes after the minimap icon then crashes. Summary crash log says 'The game crashed whilst exception in server tick loop
Error: java.lang.NoClassDefFoundError: sereneseasons/config/ServerConfig' and i have no idea on how to fix it and instead revert back to the old version

### How can the issue be reproduced?

Update Serene and try loading or creating a world

### Logs

https://gist.github.com/Clyd3Th3R4bb1t/a6f24d057820656ba30f033043e41421

### Mod Version

9.1.0.0

### Additional information

It Seems That it is a code issue

**Comment by gabepikamole** [comment]
same issue here! can provide crash report too, if needed. something related to snow! real magic, my crash report states.

**Comment by deltaminimal** [comment]
Same here!

**Comment by Adubbz** [comment]
The crash is due to dynamic trees relying on a class which doesn't exist in the backported version of the mod (which wasn't done by us). You can either downgrade to an older version for 1.20.1, or alternatively Dynamic Trees will require an update

**Comment by cata023** [comment]
I'm getting the same issue, but it seems to be from Snow! Real Magic instead. I also don't have Dynamic Trees.

**Comment by deltaminimal** [comment]
Me too, I dont have Dynamic Trees and the crash seems to be related to Snow! Real Magic, Older versions of Serene Season work just fine, anything above 9.0.0.46 seems to have problems with that mod.

**Comment by Clyd3Th3R4bb1t** [comment]
> The crash is due to dynamic trees relying on a class which doesn't exist in the backported version of the mod (which wasn't done by us). You can either downgrade to an older version for 1.20.1, or alternatively Dynamic Trees will require an update

That's just weird. It Seems Downgraded versions of Serene seasons works fine with Dynamic trees though-so I'll use that until the issue gets resolved from either Serene or Dynamic trees. thanks for the explanation though.

**Comment by vinnyg0621** [comment]
same issue here

**Comment by deltaminimal** [comment]
It now works with the new Snow! Real Magic update, they fixed what was happening on their end and its now fully compatible with 9.1

**Comment by raffle2** [comment]
Same issue here on forge 1.20.1. Only works with an older version (9.0.0.46). Anything above that and the game just crashes after it finishes loading into a world. Don't have Dynamic Trees. And Snow! Real Magic is updated to latest.

**Comment by vinnyg0621** [comment]
no, it still crashes. i was just doing this in my new world and it crashed.

**Comment by vinnyg0621** [comment]
> no, it still crashes. i was just doing this in my new world and it crashed.

nvm, I updated "snow under trees"
